edge-apps/rss-reader/screenly.yml

```diff
@@ -66,3 +66,9 @@ settings:
     help_text: |-
       Determines the visual theme for the application.
       Options are 'light' or 'dark'.
+  sentry_dsn:
+    type: secret
+    title: Sentry Client Key
+    optional: true
+    help_text: Sentry Client Key from Sentry SDK for error capturing.
+    is_global: true
```

Comment:
When a new app is created via `screenly edge-app create`, the Screenly CLI app will re-arrange the fields in a certain manner (probably alphabetically). Make sure that when an Edge App gets created, fields won't get rearranged.

Reply:
Sure, Thanks for the heads-up. 

This is global setting, so will not display in the edge app store or the user.